diff --git a/local-monitors.h b/local-monitors.h
index 9939e52..ae409d7 100644
--- a/local-monitors.h
+++ b/local-monitors.h
@@ -1,3 +1,3 @@
-/* name       mfact nmaster scale layout       rotate/reflect				x	y	resx	resy	rate	adaptive	custom*/
-{ NULL,       0.55, 1,      1,    &layouts[0], WL_OUTPUT_TRANSFORM_NORMAL,	0,	0,	0,		0,		0,		0,			0},
+/* name       mfact nmaster scale	layout       rotate/reflect				x	y	resx	resy	rate	adaptive	custom*/
+{ NULL,       0.55, 1,      1.5,	&layouts[0], WL_OUTPUT_TRANSFORM_NORMAL,	0,	0,	0,		0,		0,		0,			0},
 
